micropip/package_manager.py

```diff
@@ -4,6 +4,7 @@
 )
 
 from micropip import package_index
+from micropip._commands.freeze import _freeze
```

Comment:
I don't like `_commands` directory being imported from parent codes. So let's move `_freeze`, `freeze_data` functions to a separate file (e.g. `micropip/freeze.py`) and let both `_commands/freeze.py` and `packagee_manger.py` import from it.